### PR TITLE
feat: Card Component

### PR DESCRIPTION
**Pull request by ozairkhan2810**
This PR contains six Card types

- Card with badge
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Vertical/Horizontal cards
- Cards with shadow

link: https://practical-nightingale-e88858.netlify.app

### CODE REVIEW

File: documentation/index.html [hunk 1]
```diff
@@ -131,9 +184,158 @@ <h3>Button</h3>
             <button class="btn btn-float btn-flex btn-pt-1">+</button>
```
Comment (barbaricankit) [inline documentation/index.html]: padding has not anything to do with button, so you could name simply pt-1

File: documentation/index.html [hunk 2]
```diff
@@ -131,9 +184,158 @@ <h3>Button</h3>
...
+                class="img-card"
+                src="../assets/cricket/ipl.jpg"
+                alt="card-pic"
+              />
+              <div class="card-badge-msg">Coming Soon</div>
+            </div>
+            <div class="card-main">
+              <h3 class="card-title">IPL - Indian Premier League</h3>
+              <h5 class="card-by">by Ozair Khan</h5>
+            </div>
```
Comment (barbaricankit) [inline documentation/index.html]: give proper class name

File: documentation/index.html [hunk 3]
```diff
@@ -131,9 +184,158 @@ <h3>Button</h3>
...
+          <p class="text-content">Card with dismiss</p>
+          <div class="card-dismiss">
+            <div class="rel">
+              <img
+                class="img-card"
+                src="../assets/cricket/ipl.jpg"
+                alt="card-pic"
+              />
+              <div class="card-dismiss-pic">
+                <img src="../assets/card/cross.svg" alt="" />
```
Comment (barbaricankit) [inline documentation/index.html]: give an alt name

File: documentation/index.html [hunk 4]
```diff
@@ -131,9 +184,158 @@ <h3>Button</h3>
...
+          <div class="card-text-overlay">
+            <div class="rel">
+              <img
+                class="img-card"
+                src="../assets/cricket/ipl.jpg"
+                alt="card-pic"
+              />
+              <div class="card-text-overlay-main">
+                <h3 class="card-title">IPL - Indian Premier League</h3>
+                <h5 class="card-by">by Ozair Khan</h5>
```
Comment (barbaricankit) [inline documentation/index.html]: same as above

File: lib/components/card.css [hunk 5]
```diff
@@ -0,0 +1,83 @@
+.card-badge,
+.card-dismiss,
+.card-text-overlay,
+.card-text-only,
+.card-horizontal,
+.card-vertical,
+.card-shadow {
+  display: flex;
+  flex-direction: column;
+  width: 18rem;
```
Comment (barbaricankit) [inline lib/components/card.css:10]: great using comma-separated for common properties

**Review by barbaricankit — COMMENTED** [review]
Please look into the comments I have suggested. If you have any questions please let me know.
Class names can be more proper, using name according to the styles you are using would be more meaningful

File: lib/components/card.css (hunk 5 above)
Comment (ozairkhan2810) [reply lib/components/card.css:10]: Thanks!
Will update as suggested.

**Comment by barbaricankit** [comment]
Looks good to be merged

**Review by barbaricankit — APPROVED** [review]
Looks good to merge.

**Comment by tushartiwari7** [comment]
code looks neat. you can go ahead and merge this.

**Review by tushartiwari7 — APPROVED** [review]

**Pull request merged**